### ISSUE TITLE
Clips4Sale not scraping correctly

### ISSUE THREAD

**Issue by innersix** [PAhelper/PhoenixAdult.bundle#1186]
1. Scene URL: [Clips4sale.com/…](https://www.clips4sale.com/studio/7373/427276/non-stop-blowjob)
2. File Name: `Clips4Sale - 7373 - Non-stop blowjob.wmv`
3. Log File: (see further down)

### Gist from the HTTP log:

```
< Target URL: "https://www.clips4sale.com/studio/7373/*/Cat0-AllCategories/Page1/SortBy-bestmatch/Limit50/search/Klixen%20Non%20Stop%20Blowjob"

< GET /studio/7373/*/Cat0-AllCategories/Page1/SortBy-bestmatch/Limit50/search/Klixen%20Non%20Stop%20Blowjob HTTP/1.1
< Host: www.clips4sale.com

> HTTP/1.1 301 Moved Permanently
> age: 0
> content-type: text/html; charset=UTF-8
> date: Wed, 03 Feb 2021 18:56:17 GMT
> location: https://www.clips4sale.com/studio/7373/k-handjob-by-klixen-/Cat0-AllCategories/Page1/SortBy-bestmatch/Limit50/search/Klixen Non Stop Blowjob
```

The page of the `location` header does indeed display the correct clip as the sole search result, so I assume it just should follow the 301, but doesn't?

### Log File
```
2021-02-03 19:02:48,355 (-4fa01ba0) :  INFO (agentkit:961) - Searching for matches for {'openSubtitlesHash': 'b937df0a2641176b', 'name': 'Clips4sale 7373 Non Stop Blowjob', 'filename': '%2Fmedia%2Fwdpassport1tb%2FAdult%2FVideos%2FClips4Sale%20-%207373%20-%20Non-stop%20blowjob%2Ewmv', 'plexHash': '2c77e73ae291155e9b33c3c30b361328576fb77f', 'duration': '469440', 'id': '1158'}
2021-02-03 19:02:48,357 (-4fa01ba0) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/1158/tree'
2021-02-03 19:02:48,378 (-4fa01ba0) :  INFO (__init__:62) - ***MEDIA TITLE [from media.name]*** Clips4sale 7373 Non Stop Blowjob
2021-02-03 19:02:48,379 (-4fa01ba0) :  INFO (PAsearchSites:52) - mediaTitle w/ possible abbreviation: Clips4sale 7373 Non Stop Blowjob
2021-02-03 19:02:48,434 (-4fa01ba0) :  INFO (PAsearchSites:60) - mediaTitle w/ possible abbrieviation fixed: Clips4sale 7373 Non Stop Blowjob
2021-02-03 19:02:48,455 (-4fa01ba0) :  INFO (PAsearchSites:45) - Site found
2021-02-03 19:02:48,456 (-4fa01ba0) :  INFO (PAsearchSites:72) - ^^^^^^^ siteNum: 760
2021-02-03 19:02:48,456 (-4fa01ba0) :  INFO (PAsearchSites:73) - ^^^^^^^ Shortening Title
2021-02-03 19:02:48,460 (-4fa01ba0) :  INFO (PAsearchSites:96) - Search Title (before date processing): 7373 Non Stop Blowjob
2021-02-03 19:02:48,462 (-4fa01ba0) :  INFO (__init__:64) - {'siteName': 'clips4sale', 'searchDate': None, 'siteNum': 760, 'searchTitle': '7373 Non Stop Blowjob'}
2021-02-03 19:02:48,462 (-4fa01ba0) :  INFO (PAsearchData:17) - SearchData.title: 7373 Non Stop Blowjob
2021-02-03 19:02:48,463 (-4fa01ba0) :  INFO (PAsearchData:23) - SearchData.filename: Clips4Sale - 7373 - Non-stop blowjob
2021-02-03 19:02:48,463 (-4fa01ba0) :  INFO (__init__:92) - Provider: siteClips4Sale
2021-02-03 19:02:48,467 (-4fa01ba0) :  INFO (PAutils:107) - Requesting GET "https://www.clips4sale.com/studio/7373/*/Cat0-AllCategories/Page1/SortBy-bestmatch/Limit50/search/Non%20Stop%20Blowjob"
2021-02-03 19:02:50,225 (-4fa01ba0) :  INFO (PAutils:210) - GZip request saved as "842d7eaf17ff4288bab8f455e9fcf508.gz"
2021-02-03 19:02:50,259 (-4fa01ba0) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2021-02-03 19:02:50,260 (-4fa01ba0) :  DEBUG (runtime:924) - Response: [200] str, 320 bytes
```

[842d7eaf….gz](https://github.com/PAhelper/PhoenixAdult.bundle/files/5920880/842d7eaf17ff4288bab8f455e9fcf508.gz)

**Comment by DirtyRacer1337** [comment]
Fixed in 86a01f07c8cded00cddb8ca3ee0dfd6cd3925499

**Comment by innersix** [comment]
just for me to learn; so it's not that it didn't follow the 301 but it was a scraping problem? Did I miss something in the logs then?
